tests/core/_orchestrator/test_orchestrator__submit.py

```diff
@@ -54,18 +53,16 @@ def test_submit_scenario_development_mode():
     scenario.dn_0.write(0)  # input data is made ready
     orchestrator = _OrchestratorFactory._build_orchestrator()
 
-    submit_time = datetime.now() + timedelta(seconds=1)  # +1 to ensure the edit time of dn_0 is before the submit time
-    with freezegun.freeze_time(submit_time):
-        submission = orchestrator.submit(
-            scenario, no_of_retry=10, log=True, log_file="file_path"
-        )  # scenario is executed directly in development mode
-        jobs = submission.jobs
+    submit_time = datetime.now()
+    submission = orchestrator.submit(
+        scenario, no_of_retry=10, log=True, log_file="file_path"
```

Comment:
what are these properties for? If we want to test the custom properties mechanism, we should add some assertions, and it should be in a different test function.

Reply:
Idk. It was there before

Reply:
They are used below. Let s keep them. Thx.